server/index.js

```diff
@@ -20,5 +21,81 @@ app.get('/api/alsoBought/:id', (req, res) => {
   });
 });
 
+// Get all stock records
+app.get('/api/alsoBought/', (req, res) => {
+  Stock.readAll((err, stocks) => {
+    if (err) {
+      res.sendStatus(400);
+    } else {
+      res.status(200).json(stocks);
+    }
+  });
+});
+
+//Show the stock record in the db for the specific id
+app.get('/api/alsoBought/:id/show', (req, res) => {
+  Stock.readOne(req.params.id, (err, stock) => {
+    if (stock) {
+      res.status(200).json(stock);
+    } else {
+      res.sendStatus(404);
```

Comment:
Why do you use res.send in some places and res.sendStatus in others? Might choose one and stick to it, for consistency. 

Also, would try to keep the error handling structure consistent across routes. I think it is better to check for an error  in the beginning of your if statement, because that way you are always checking for an error object. Here you are checking for the truthiness of stock, which isn't quite the same thing.

Reply:
The second comment also applies to line 60

Reply:
I'll made this change.